libs/components/src/toggle-group/toggle-group-button.component.ts

```diff
@@ -0,0 +1,82 @@
+import { HostBinding, Component, Input } from "@angular/core";
+
+import { ToggleGroupComponent } from "./toggle-group.component";
+
+let nextId = 0;
+
+@Component({
+  selector: "bit-toggle-group-button",
+  templateUrl: "./toggle-group-button.component.html",
+  preserveWhitespaces: false,
+})
+export class ToggleGroupElementComponent {
+  id = nextId++;
+
+  @Input() value?: string;
+
+  constructor(private groupComponent: ToggleGroupComponent) {}
+
+  @HostBinding("tabIndex") tabIndex = "-1";
+  @HostBinding("class") get classList() {
+    return ["tw-group"];
+  }
```

Comment:
I think we can change this to just be variable.

```suggestion
  @HostBinding("class") classList = ["tw-group"];
```